### PR TITLE
Fix: pom amendments

### PR DESCRIPTION
**Pull request by matthewandrewpalmer**
Refactoring of POM:

- Moved versions number to properties section
- Moved  plugin configuration to plugin management

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -84,39 +87,43 @@
...
-        </executions>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.felix</groupId>
-        <artifactId>maven-bundle-plugin</artifactId>
       </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.14.1</version>
```
Comment (ajorpheus) [inline pom.xml]: Would it be possible to move this over to pluginManagement as well please?

**Review by ajorpheus — CHANGES_REQUESTED** [review]
Looks good, just one minor recommendation

File: pom.xml [hunk 2]
```diff
@@ -84,38 +87,46 @@
             <target>${java.version}</target>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>com.github.ekryd.sortpom</groupId>
+          <artifactId>sortpom-maven-plugin</artifactId>
+          <version>2.8.0</version>
```
Comment (ajorpheus) [inline pom.xml]: Move this version to properties section please?

File: pom.xml [hunk 3]
```diff
@@ -84,38 +87,46 @@
...
+                <goal>sort</goal>
+              </goals>
+              <phase>verify</phase>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>2.14.1</version>
```
Comment (ajorpheus) [inline pom.xml]: Move this version to properties section please?

**Review by ajorpheus — CHANGES_REQUESTED** [review]

**Review by ajorpheus — DISMISSED** [review]
I have added some changes around the sortpom plugin configuration.

Looks good to me.

**Review by ajorpheus — DISMISSED** [review]
Awaiting comment from Matt about results of a local build and deploy test.

**Comment by matthewandrewpalmer** [comment]
Built successfully and services running correctly when deployed in CDAP.

Returning JSON of company house:
image

**Review by ajorpheus — APPROVED** [review]

**Pull request merged**